spec/index.bs

```diff
@@ -17162,10 +17162,7 @@ The [=texel block memory cost=] of each of these formats is the same as its
         <td>{{GPUTextureFormat/bgra8unorm-srgb}}
         <td>
         <td>{{GPUTextureSampleType/"float"}},<br/>{{GPUTextureSampleType/"unfilterable-float"}}
-        <td>&checkmark;
-        <td>&checkmark;
-        <td>&checkmark;
-        <td>&checkmark;
+        <td colspan=4>If {{GPUFeatureName/"core-features-and-limits"}} is enabled
```

Comment:
This would still allow sampling, we should probably use the "Required Feature" column.